src/input-capture.c

```diff
@@ -222,10 +289,6 @@ validate_capabilities (const char  *key,
   return TRUE;
 }
 
-static XdpOptionKey input_capture_create_session_options[] = {
```

Comment:
Why is this PR open coding this instead?

Reply:
`xdp_filter_options()` filters and appends to the builder but there's only one option (`capabilities`) and the ones we pass on are the possibly modified capabilities, not the original ones. So we'd have to build a new options variant or somehow overwrite the existing one in the builder, both seem more work.

Although now that I write this I wonder if `g_variant_builder_add()` overwrites the key-value if it already exists but from a quick test case I tried here it just takes the first value added and ignores subsequent ones.